Makefile

```diff
@@ -8,19 +8,19 @@ GIT_BRANCH := $(shell git symbolic-ref -q --short HEAD)
 GIT_VERSION := $(shell git log -1 --pretty=format:"%h (%ci)")
 
 build:
-	go build -ldflags "-X \"main.buildVersion=$(GIT_VERSION)\"" \
+	GO111MODULE=on go build -ldflags "-X \"main.buildVersion=$(GIT_VERSION)\"" \
```

Comment:
Hm, I had noticed that approach did not work for other repos. I made a test:

```makefile
GO111MODULE ?= on

all:
	@@env | grep GO111 || echo 'nope'
	@@GO111MODULE=on env | grep GO111 || echo 'nope'
```

The first call echos 'nope'.. while the second echo the presence of the variable.

Reply:
OK, that was only working for me because I had `GO111MODULE=on` exported in my shell. If you change the line to `export GO111MODULE ?= on` it should work and does in my tests (after unsetting the variable in my shell).

Reply:
Ah ok thanks.